crates/sui-protocol-config/src/lib.rs

```diff
@@ -2589,6 +2595,14 @@ impl ProtocolConfig {
     pub fn set_mysticeti_num_leaders_per_round_for_testing(&mut self, val: Option<usize>) {
         self.feature_flags.mysticeti_num_leaders_per_round = val;
     }
+
+    pub fn set_enable_soft_bundle(&mut self, val: bool) {
+        self.feature_flags.soft_bundle = val;
+    }
+
+    pub fn set_max_soft_bundle_size(&mut self, val: u64) {
```

Comment:
You shouldn't need this function, a macro is already generating test version of this function for all flags.